src/main/kotlin/net/minecrell/pluginyml/bukkit/BukkitPlugin.kt

```diff
@@ -60,6 +60,10 @@ class BukkitPlugin : PlatformPlugin<BukkitPluginDescription>("Bukkit", "plugin.y
                 if (alias.contains(':')) throw InvalidPluginDescriptionException("Alias '$alias' of '${command.name}' cannot contain ':'")
             }
         }
+
+        if (description.provides?.all(VALID_NAME::matches)?.not() == true) {
```

Comment:
What is the `?.not()` here? Why not just

```suggestion
        if (description.provides?.all(VALID_NAME::matches) == false) {
```